src/tests/ftest/util/nvme_utils.py

```diff
@@ -44,19 +44,19 @@ def set_device_faulty(test, dmg, server, uuid, pool=None, has_sys_xs=False, **kw
     Args:
         test (Test): avocado test class
         dmg (DmgCommand): a DmgCommand class instance
-        server (NodeSet): host on which to issue the dmg storage set nvme-faulty
+        server (NodeSet): host on which to issue the dmg storage set nvme-faulty. Must be one host.
         uuid (str): the device UUID
         pool (TestPool, optional): pool used to wait for rebuild to start/complete if specified.
             Defaults to None.
         has_sys_xs (bool, optional): the device's has_sys_xs property value. Defaults to False.
         kwargs (dict, optional): named arguments to pass to the DmgCommand.storage_set_faulty.
 
     Returns:
-        dict: the json response from the dmg storage set-faulty command.
-
+        dict: the json response from the dmg storage set-faulty command. None if has_sys_xs is True.
     """
     kwargs['host'] = server
     kwargs['uuid'] = uuid
+    response = None
     try:
         response = get_dmg_response(dmg.storage_set_faulty, **kwargs)
```

Comment:
This should cover having to verify the None return value:
```suggestion
    response = None
    try:
        response = get_dmg_response(dmg.storage_set_faulty, **kwargs)
        if has_sys_xs:
            test.fail("Setting a sys_xs device faulty should fail.")
```